lib/apt_news.py

```diff
@@ -22,17 +24,18 @@ def main(cfg: UAConfig):
 
 
 if __name__ == "__main__":
+    root_logger.propagate = False
     setup_logging(
         logging.INFO,
         logging.DEBUG,
         defaults.CONFIG_DEFAULTS["log_file"],
-        logger=logging.getLogger(),
+        logger=root_logger,
     )
```

Comment:
I think we should change `setup_logging` to always configure the "uaclient" logger rather than accept it as an argument.

Reply:
Valid point. I have gone thru' the various instances where we call `setup_logging`, most are calling `uaclient_logger` .

Reply:
After the discussion on standup, I will replace all calls to cli.setup_logging and then later replace daemon.setup_logging with the logging to the journald.